lib/entities/state/attributes/DoNotDisturbAttribute.js

```diff
@@ -0,0 +1,23 @@
+const StateAttribute = require("./StateAttribute");
+
+class DoNotDisturbAttribute extends StateAttribute {
```

Comment:
This shouldn't be a State Attribute but instead called `ValetudoDNDConfiguration` which extends `SerializableEntity` and would be found in `lib/entities/core`.

There you can also find the `ValetudoWifiConfiguration` class, which does the same thing but for wifi